options/locale/locale_en-US.ini

```diff
@@ -3546,6 +3544,10 @@ variables.creation.success = The variable "%s" has been added.
 variables.update.failed = Failed to edit variable.
 variables.update.success = The variable has been edited.
 
+[artifacts]
+title = Artifacts
+delete_confirm = Are you sure?
+
```

Comment:
Not sure about this change, it will invalidate all existing translations and require re-translation on Crowdin. Changing it in existing locales does not work in our translation system. So I guess revert this. `delete_confirm` is generic enough to not be needed under the `artifacts` section and is universally useful to have, so put it on top level as well.

Reply:
reverted